src/riak_kv_pb_timeseries.erl

```diff
@@ -316,21 +321,28 @@ put_data(Data, Table, Mod) when is_binary(Table) ->
                   case riak_kv_w1c_worker:validate_options(
                          NVal, Preflist, [], BucketProps) of
                       {ok, W, PW} ->
-                          {Ids, Errs} =
-                              lists:foldl(
-                                fun(Record, {PartReqIds, PartErrors}) ->
-                                        {RObj, LK} =
-                                            build_object(Bucket, Mod, DDL,
-                                                         Record, DocIdx),
-
-                                        {ok, ReqId} =
-                                            riak_kv_w1c_worker:async_put(
-                                              RObj, W, PW, Bucket, NVal, LK,
-                                              EncodeFn, Preflist),
-                                        {[ReqId | PartReqIds], PartErrors}
-                                end,
-                                {[], 0}, Records),
-                          {GlobalReqIds ++ Ids, GlobalErrorsCnt + Errs};
+                          DataForVnode = pick_batch_option(SendFullBatches,
```

Comment:
That's toodamnhigh an indentation. Can you do something to reduce the vast white space on the left?